src/graph-model.js

```diff
@@ -34,5 +34,6 @@ export default class GraphModel {
       }
       Object.defineProperty(this, key, descriptor);
     });
+    Object.defineProperty(this, 'attrs', { enumerable: false });
```

Comment:
Is https://github.com/Shopify/graphql-js-client/pull/90/files/7beb6aebeb80213944d496ded3d041ba263cae34..c407ff28bd8b46bc5cee9adc631db4c914deb5e7#diff-fe88f4dbb0d91d2356b6fa5e8f437e85R13 still needed? Should this line be first in the constructor?